packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -280,6 +285,9 @@ export default class ImageEdit implements EditorPlugin {
                 this.removeWrapper(wrapper);
             }
 
+            // Keep Image responsive, if not resized
+            setImageResponsive(this.image, this.wasResized);
```

Comment:
this logic should be part of applyChange(). We set image width/height there, so we should change it according to the value of wasResized.